eng/common/scripts/Delete-RemoteBranches.ps1

```diff
@@ -1,38 +1,71 @@
 param(
+  # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
+  [Parameter(Mandatory = $true)]
+  $RepoNameWithOwner,
+  # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
   [Parameter(Mandatory = $true)]
   $RepoOwner,
+  # Please use the RepoOwner/RepoName format: e.g. Azure/azure-sdk-for-java
+  [Parameter(Mandatory = $true)]
+  $RepoName,
   # Use this if a pull request might have been opened from one repo against another.
   # E.g Pull request opened from azure-sdk/azure-sdk prBranch --> Azure/azure-sdk baseBranch
   $ForkRepoOwner,
   [Parameter(Mandatory = $true)]
-  $RepoName,
-  [Parameter(Mandatory = $true)]
-  $BranchPrefix,
+  $BranchPrefix,  
+  # Date format: e.g. Tuesday, April 12, 2022 1:36:02 PM. Allow to use other date format.
+  $LastCommitOlderThan=$env:LASTCOMMITDATEOVERRIDE,
   [Parameter(Mandatory = $true)]
   $AuthToken
 )
 
 . (Join-Path $PSScriptRoot common.ps1)
 
-LogDebug "Operating on Repo [ $RepoName ]"
+function GetCommitDate($commitUrl) {
```

Comment:
Why is this not added to Invoke-GitHubAPI helpers?

Reply:
It is a little different than other call. But it is still related to Github operation. I moved to Invoke-GitHubAPI.